static/src/javascripts/es6/projects/common/modules/crosswords/helpers.js

```diff
@@ -1,40 +1,30 @@
-// vendor
 import _ from 'common/utils/_';
 
-// app
 import constants from 'es6/projects/common/modules/crosswords/constants';
 
-function isAcross(clue) {
-    return clue.direction === 'across';
-}
+let isAcross = (clue) => clue.direction === 'across';
```

Comment:
I've always preferred this syntax for declaring/assigning functions, just for simplicity. Function declarations are hoisted (with assignment as well as declaration) but I've never found that appealing.